src/openapi/transport/retry.js

```diff
@@ -150,12 +171,31 @@ TransportRetry.prototype.retryFailedCalls = function() {
 };
 
 /**
- * Disposes the underlying transport, the failed calls queue and clears retry timer.
+ * @protected
+ * @param transportCall
+ */
+TransportRetry.prototype.retryIndividualFailedCall = function(transportCall) {
+    transportCall.retryTimer = null;
+    const individualFailedCallsIndex = this.individualFailedCalls.indexOf(transportCall);
+    if (individualFailedCallsIndex >= 0) {
+        this.individualFailedCalls.splice(individualFailedCallsIndex, 1);
+    }
+    this.sendTransportCall(transportCall);
+};
+
+/**
+ * Disposes the underlying transport, the failed calls queue and clears retry timers.
  */
 TransportRetry.prototype.dispose = function() {
+    this.individualFailedCalls.forEach((failedCall) => {
+        if (failedCall.retryTimer) {
+            failedCall.retryTimer.clearTimeout();
+            failedCall.retryTimer = null;
+        }
+    });
     this.failedCalls.length = 0;
     if (this.retryTimer) {
-        this.retryTimer.clear();
+        this.retryTimer.clearTimeout();

```

Comment:
No you're right... it should be clearTimeout(this.retryTimer), I don't know what made me think it's an object, and apparently this.retryTimer.clear() wasn't doing much. Basically dispose was and still is broken, I guess it's not critical but worth fixing for next version